andela/colossus-backend
# #167733846: Accommodation Facility

**What does this PR do?**

Enables users to create accommodation services and others to book them

**Description of the task to be completed**

A user should be able to book an accommodation and indicate duration of stay

**How should this be manually tested?**

Hit the route '/api/v1/accommodation/book' with Postman or equivalent. 
But before that, an accommodation service should be created with '/api/v1/accommodation/create' and a multipart body should be sent. It should contain the fields "type" and "picture" which are string and file types respectively.
To book an accommodation, send a JSON body with the fields "movingIn" and "movingOut" which are dates the user would move in and out of the accommodation respectively.

**Any background context that you want to provide?**

N/A

**What are the relevant pivotal tracker stories?**

[#167733846](https://www.pivotaltracker.com/story/show/167733846)
